.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.0
 jobs:
   test:
+    resource_class: large
```

Comment:
Previous setup was good for detecting RC. I am not sure whether we want to use more CPUs :thinking: 

Also, it looks like `resource_class` requires CircleCI configuration in version 2.1

https://circleci.com/docs/configuration-reference#executors-requires-version-21

Reply:
> Also, it looks like resource_class requires CircleCI configuration in version 2.1

This isn't true, see https://circleci.com/docs/configuration-reference#resourceclass
The thing you linked relates to executors - with 2.1 you can define an executor that can be shared between jobs

Anyway, this was just done as a suggestion from CircleCI and doesn't bring as much improvement to be worth it.